vic/drivers/classic/src/read_initial_model_state.c

```diff
@@ -42,29 +42,33 @@ read_initial_model_state(FILE            *init_state,
                          int              Nveg,
                          int              Nbands,
                          int              cellnum,
-                         soil_con_struct *soil_con)
+                         soil_con_struct *soil_con,
+                         lake_con_struct  lake_con)
 {
-    extern option_struct options;
+    extern option_struct     options;
```

Comment:
`uncrustify` should take care of this. Make sure the code is `uncrustified` before it gets merged.